extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -1762,7 +1766,8 @@ function constructSRARequest_(a4a, instances) {
         Object.assign(
           blockParameters,
           googPageLevelParameters,
-          instances[0].getPageParameters(instances[0].consentState, instances)
+          instances[0].getPageParameters(
+            instances[0].consentState, instances, consentString)
```

Comment:
No need to pass consentString through like this, instead store on the instance object and access in same manner as currently getting consentState